projects/1.19/assets/beneath-the-wetlands/wetlands-/lang/zh_cn.json

```
@@ -0,0 +1,78 @@
+{
+  "entity.wetlands.elderswampgolem":"上古铜傀儡",
+  "entity.wetlands.copperswampgolem":"铜傀儡",
+  "entity.wetlands.willothewisp":"小精灵",
+
+  "item.wetlands.raw_bog_iron":"原始沼泽铁",
+  "item.wetlands.bog_iron_ingot":"沼泽铁锭",
+  "item.wetlands.sulfur_nugget":"硫矿块",
+  "item.wetlands.bottled_willothewisp":"瓶装的小精灵",
+  "item.wetlands.willothewisp_spawn_egg":"鬼火刷怪蛋",
+  "item.wetlands.elder_golem_spawn_egg":"上古铜傀儡生成蛋",
+  "item.wetlands.copper_golem_spawn_egg":"铜傀儡刷怪蛋",
+
+  "block.wetlands.raw_bog_iron_block":"原始沼泽铁块",
+  "block.wetlands.bog_iron_ore":"沼泽铁矿",
+  "block.wetlands.cut_bog_iron_slab":"切割沼泽铁板",
+  "block.wetlands.cut_bog_iron_stairs":"切割沼泽铁楼梯",
+  "block.wetlands.bog_iron_block":"沼泽铁块",
+  "block.wetlands.cut_bog_iron":"切割沼泽铁",
+
+
+  "block.wetlands.silt":"淤泥",
+  "block.wetlands.peat":"泥炭",
+  "block.wetlands.dry_peat":"干泥炭",
+  "block.wetlands.copper_gear":"铜齿轮",
+  "block.wetlands.animated_copper_gear":"动画铜齿轮",
+
+  "wetlands.swampcall4": "沼泽的呼唤",
+  "wetlands.swampcall3": "沼泽的呼唤",
+  "wetlands.swampcall2": "沼泽的呼唤",
+  "wetlands.swampcall1": "沼泽的呼唤",
+  "wetlands.coppergolemattack": "铜魔像攻击",
+  "wetlands.coppergolemdeath": "铜魔像死亡",
+  "wetlands.coppergolemhurt": "铜魔像受伤",

```

```suggestion
  "wetlands.coppergolemattack": "铜魔像：攻击",
  "wetlands.coppergolemdeath": "铜魔像：死亡",
  "wetlands.coppergolemhurt": "铜魔像：受伤",
```
根据 Minecraft Wiki 上的[字幕](https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95)的翻译习惯，须在实体与动作之间加上：（注意是全角冒号）
原文：Bee buzzes（[Minecraft](https://minecraft.fandom.com/wiki/Subtitles), Mojang）
译文：蜜蜂：嗡嗡